diff --git a/dist/index.cjs b/dist/index.cjs
index 5636310421ab08046e94bd9c7f2b338a663d99b6..15d2ac8d8c90db72cb027c96d6184ba2eb04973f 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -444,6 +444,7 @@ var YoutubeExtractor = class extends discord_player.BaseExtractor {
 		}
 	}
 	async deactivate() {
+		tube = null;
 		this.innertube = null;
 		this._stream = null;
 		if (this.interval) clearInterval(this.interval);
diff --git a/dist/index.mjs b/dist/index.mjs
index 7982bb477b68b080e7e0d305ec9a9e129fb29843..4af68a162d209361a03697aeab47a4ee668f612a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -420,6 +420,7 @@ var YoutubeExtractor = class extends BaseExtractor {
 		}
 	}
 	async deactivate() {
+		tube = null;
 		this.innertube = null;
 		this._stream = null;
 		if (this.interval) clearInterval(this.interval);
